### ISSUE TITLE
[Bug]: swipeBehaviour.disableVertical() doesn't work

### ISSUE THREAD

**Issue by daniilkananenka** [Telegram-Mini-Apps/telegram-apps#490]
### Telegram Application

Telegram for Android, Telegram for iOS

### Describe the Bug

In new major version of @telegram-apps/sdk-react `swipeBehavior.disableVertical()` doesn't work as expected - it doesn't disable vertical swipes at all.

### To Reproduce

1. Install new major version of @telegram-apps/sdk-react
2. In your `init.ts` file add `import {swipeBehavior} from '@telegram-apps/sdk-react`
3. In your `init` function add 
```ts
swipeBehavior.mount();
swipeBehavior.disableVertical();
```
4. Run your mini app on mobile with Telegram App, that supports vertical behavior
5. Try to close mini app using swipe and see error

### Expected Behavior

`swipeBehavior.disableVertical()` should disable vertical swipes

**Comment by daniilkananenka** [comment]
Temporarily fix from @heyqbnk :
```ts
postEvent('web_app_setup_swipe_behavior', { allow_vertical_swipe: false });
```

**Comment by heyqbnk** [comment]
Fixed in `@telegram-apps/sdk@2.4.0`. Thank you for report!

PR: https://github.com/Telegram-Mini-Apps/telegram-apps/pull/491
